scripts/style.py

```diff
@@ -165,7 +165,6 @@ def unused_imports(args, files_to_check):
     "src/transaction_sender/transaction_pool.zig",
     "src/gossip/ping_pong.zig",
     "src/accountsdb/download.zig",
-    "src/accountsdb/fuzz.zig",
     "src/ledger/reader.zig",
```

Comment:
shouldn't we be adding it in this PR?

Reply:
No, these are the files excluded from the line length check; it no longer needs to be on the excluded list since it now conforms to that line length.